contracts/profiles/StealthKeyResolver.sol

```diff
@@ -4,36 +4,64 @@ import "../ResolverBase.sol";
 abstract contract StealthKeyResolver is ResolverBase {
     bytes4 constant private STEALTH_KEY_INTERFACE_ID = 0x69a76591;
 
-    event StealthKeyChanged(bytes32 indexed node, uint256 generationPubKey, uint256 encryptionPubKey);
+    event StealthKeyChanged(bytes32 indexed node, uint256 spendingPubKeyPrefix, uint256 spendingPubKey, uint256 viewingPubKeyPrefix, uint256 viewingPubKey);
 
-    struct StealthKey {
-        uint256 generationPubKey;
-        uint256 encryptionPubKey;
-    }
-
-    mapping(bytes32=>StealthKey) _stealthKeys;
+    // node => prefix => key
+    mapping(bytes32 => mapping(uint256 => uint256)) _stealthKeys;
```

Comment:
Some explicit documentation on the mapping between each key and its prefix to the values in the array here would be good.

Reply:
Good idea, will add 👍